tests/acceptance/editor-selections-test.js

```diff
@@ -156,13 +159,14 @@ test('select text and apply markup multiple times', (assert) => {
   Helpers.dom.triggerEvent(document, 'mouseup');
 
   setTimeout(() => {
-    Helpers.toolbar.clickButton(assert, 'bold');
+    let command = new BoldCommand(editor);
+    command.exec();
 
     Helpers.dom.selectText('fir', editorElement);
     Helpers.dom.triggerEvent(document, 'mouseup');
 
     setTimeout(() => {
-      Helpers.toolbar.clickButton(assert, 'bold');
+      command.exec();
```

Comment:
there should no longer be a need to use `setTimeout` (or triggering "mouseup") in these tests.